assets/css/footer.scss

```diff
@@ -0,0 +1,150 @@
+$default-font: "Roboto", "Times New Roman", serif;
+$second-font: "Raleway", "Times New Roman", serif;
+
+$light-green: #497557;
+$green: #0c4a34;
+$grey: #757575;
+$dark-green: #02291c;
+$text-color-dark: #214e41;
+$text-color-light: #43806c;
+$white: #ffffff;
+
+$mobile: 375px;
+$tablet: 768px;
+$desktop: 1280px;
+$large-desktop: 1560px;
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+
+  width: 100%;
+  height: 80%;
+  padding-top: 5%;
+
+  background-color: $light-green;
+
+  @media (min-width: $tablet) and (max-width: $desktop) {
+    align-items: stretch;
+    padding-top: 2%;
+    height: 60%;
+  }
+  @media (min-width: $desktop) {
+    align-items: stretch;
```

Comment:
There is no need to write them a second time:
    
    align-items: stretch;
    padding-top: 2%;